explainer.md

```diff
@@ -520,6 +520,18 @@ xrSession.updateRenderState({
 });
 ```
 
+### Preventing the compositor from using the depth buffer
+
+By default the depth attachment of an `XRWebGLLayer`'s `framebuffer`, if present, may be used to assist the XR compositor. For example, the scene's depth values may be used by advanced reprojection techniques or to help avoid depth disparity when rendering platform/UA interfaces. This assumes, of course, that the values in the depth buffer are representative of the scene content.
```

Comment:
> depth disparity

Is that an official term?  I've never heard it though I have heard "depth inversion" used before...

Reply:
I'm reasonably certain that I've heard it from multiple different VR developers in one form or another, but doing a quick Google search on the phrase doesn't yield the results I would have thought, so perhaps this isn't as common as I thought. "Depth inversion" isn't a term I've ever heard applied to this specific effect, but I did come across the term "depth conflict" and that seems like a winner all around, with the term inherently suggesting "this is a bad thing that you don't want!" ;)

Reply:
> For example, the scene's depth values may be used by advanced reprojection techniques or to help avoid depth disparity when rendering platform/UA interfaces. This assumes, of course, that the values in the depth buffer are representative of the scene content.

I think this example doesn't need to be part of the spec.